docs/sources/models.md

```diff
@@ -142,6 +142,13 @@ model = keras.models.Graph()
             - __input__: str name of the node/input that the node is connected to. Only specify *one* of either `input` or `inputs`.
             - __inputs__: list of str names of the node that the node is connected to.
             - __merge_mode__: "sum" or "concat". Only applicable if `inputs` list is specified. Merge mode for the different inputs.
+    - __add_shared_node__(layer, name, inputs=[], merge_mode=None, outputs=[]):Add a shared node to `inputs`.
```

Comment:
A description of what a "shared node" is and how it is expected to behave would certainly be useful to users here.